lib/functions-custom.php

```diff
@@ -92,5 +92,20 @@ function get_actividad_num($actividad_id) {
     }
   }
 
-  return false;
-}
\ No newline at end of file
+  return 0;
+}
+
+// update activity number on post save
+function save_actividad_num( $post_id, $post, $update ) {
+
+    $post_type = get_post_type($post_id);
+
+    // If this isn't a 'actividad' post, don't update it.
+    if ( 'actividad' != $post_type ) return;
+
+    $activity_num = get_actividad_num($post_id);
```

Comment:
ah! the `save_post` action is also triggered when moving posts to the trash. so this will work ok i think